### PR TITLE
Feature/get balance

### PR DESCRIPTION
**Pull request by taylorjdawson**

### CODE REVIEW

File: src/address.js [hunk 1]
```diff
@@ -13,6 +13,20 @@ class Address {
...
   }
 
+  async getBalance(hash) {
+    const response = await this.getBalanceLatest(hash)
+    throwIf(
+      !response || (response.status === 200 && !response.payload),
+      'error with request'
+    )
+    let balance
+    if (response.status !== 404) {
```
Comment (TrevorJTClarke) [inline src/address.js]: @taylorjdawson what happens when 404? Seems like a dead end

File: src/web3data.js [hunk 2]
```diff
@@ -94,6 +94,42 @@ class Web3Data {
...
+
+  getTransactionFromBlock(id, index) {
+    return this.block.getTransactionFromBlock(id, index)
+  }
+
+  getTransactions(id, filterOptions) {
+    return this.block.getTransactions(id, filterOptions)
+  }
+
+  getBalance(hash) {
```
Comment (TrevorJTClarke) [inline src/web3data.js]: @taylorjdawson move under `.eth`

File: src/address.js [hunk 3]
```diff
@@ -83,7 +97,7 @@ class Address {
     })
   }
 
-  getBalance(hash, filterOptions) {
+  getBalanceLatest(hash, filterOptions) {
```
Comment (TrevorJTClarke) [inline src/address.js]: @taylorjdawson why `getBalanceLatest`?

I thought we were conforming to ethers & web3?

File: src/constants.js [hunk 4]
```diff
@@ -19,5 +21,20 @@ module.exports.ERROR_MESSAGE_TRANSACTION_NO_HASH =
...
-module.exports.ERROR_MESSAGE_BLOCK_NO_NUMBER = 'No block number supplied'
+module.exports.ERROR_MESSAGE_BLOCK_NO_ID = 'No block number or hash supplied'
 module.exports.ERROR_MESSAGE_SIGNATURE_NO_HASH = 'No signature hash supplied'
+
+/* Ethereum based methods */
+module.exports.ETH_METHODS = {
+  getBlockNumber: 'block',
+  getBlock: 'block',
+  getBlockTransactionCount: 'block',
+  getUncle: 'block',
```
Comment (TrevorJTClarke) [inline src/constants.js]: @taylorjdawson `getBalance` method needs to be here, right?

Comment (taylorjdawson) [reply src/constants.js]: added!

**Pull request merged**